src/nodes/lighting/ShadowNode.js

```diff
@@ -56,7 +56,7 @@ const getShadowMaterial = ( light ) => {
 		material = new NodeMaterial();
 		material.colorNode = vec4( 0, 0, 0, 1 );
 		material.depthNode = depthNode;
-		material.isShadowNodeMaterial = true; // Use to avoid other overrideMaterial override material.colorNode unintentionally when using material.shadowNode
+		material.isShadowPassMaterial = true; // Use to avoid other overrideMaterial override material.colorNode unintentionally when using material.shadowNode
```

Comment:
The PR renames the type property to `isShadowPassMaterial` since `isShadowNodeMaterial` is reserved by `ShadowNodeMaterial`.